activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
You can use various forms of 'git reset' to alter your local git repo.  Lots of good examples on stackexchange you can find with google.  And, yes, lots of people run into issues like this.